contracts/context-config/tests/sandbox.rs

```diff
@@ -931,19 +919,21 @@ async fn test_deploy() -> eyre::Result<()> {
     let context_public = context_secret.verifying_key();
     let context_id = context_public.to_bytes().rt()?;
 
+    // Fund node1 just for gas fees
     drop(
         root_account
-            .transfer_near(node1.id(), NearToken::from_near(100))
+            .transfer_near(node1.id(), NearToken::from_near(500))
             .await,
     );
 
-    // Also transfer NEAR to the contract to cover proxy deployment costs
+    // Fund the contract with enough NEAR for proxy deployments
     drop(
         root_account
-            .transfer_near(contract.id(), NearToken::from_near(10))
+            .transfer_near(contract.id(), NearToken::from_near(100))
```

Comment:
`create_subaccount(..).initial_balance`

Reply:
hmm, I'm not sure if I'm following this. This is to fund the Context Contract account, subaccount is created in the contract itself.